operations/helm/charts/mimir-distributed/values.yaml

```diff
@@ -1581,3 +1581,79 @@ gateway:
       - secretName: gem-gateway-tls
         hosts:
           - gateway.gem.example.com
+
+metaMonitoring:
+  grafanaAgent:
```

Comment:
Is it possible to add a parameter for monitoring namespace? I usually install my ServiceMonitor definitions in the `monitoring` namespace for the prometheus operator. Would be nice to be able to use both the prom operator and this agent at the same time.  (Maybe there's a simpler solution, let's discuss IRL).

Reply:
resolved after talking about it? you're able to provide a namespace for the grafana agent resource